unittests/include/log-mock.hpp

```diff
@@ -34,6 +39,11 @@ class Log::LogMock : public Log
     {
         return Log::logger_;
     }
+
+    auto& time_created()
```

Comment:
Shouldn't it be `const`?
```suggestion
    auto const& time_created() const
```

Reply:
Non-const to allow easy modification in tests.
I don't see a reason to have a `const` getter for this tests-specific class.